examples/layout-component/package.json

```diff
@@ -7,8 +7,9 @@
     "start": "next start"
   },
   "dependencies": {
-    "next": "*"
+    "next": "^2.0.0-beta.16"
   },
+  "babel": {},
```

Comment:
This is not good. This works because, we check for `.babelrc` existence in the app dir. Unless we add `next/babel` as the preset by default.

We have to fix it. When we do that, it'll break apps using this.